backend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mhart/alpine-node:10.7.0 as test
+WORKDIR /usr/src/app
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install
+COPY . .
+RUN yarn test
```

Comment:
does it fail the build if the tests fail? We also already have our tests run in the ci

Reply:
yep, and it should fail if the tests are not passing as we don't want to deploy it in that case.